src/helpers/wincher-helper.php

```diff
@@ -14,23 +13,55 @@
  */
 class Wincher_Helper {
 
+	/**
+	 * Holds the Options Page helper instance.
+	 *
+	 * @var Options_Helper
+	 */
+	protected $options;
+
+	/**
+	 * Options_Helper constructor.
+	 *
+	 * @param Options_Helper $options The options helper.
+	 */
+	public function __construct( Options_Helper $options ) {
+		$this->options = $options;
+	}
+
 	/**
 	 * Checks if the integration is active for the current user.
 	 *
-	 * @return bool Whether or not the integration is active.
+	 * @return bool Whether the integration is active.
 	 */
 	public function is_active() {
-		// If the integration is disabled, Wincher should not be active.
-		$conditional = new Wincher_Conditional();
+		return ! $this->is_disabled() && $this->options->get( 'wincher_integration_active', true );
```

Comment:
You left out the `(bool)` that the original had. Are you sure we can never get a non boolean value from the options?